python/cudf/cudf/tests/test_csv.py

```diff
@@ -834,7 +834,7 @@ def test_csv_reader_bools_NA():
     expected = pd.DataFrame(
         {
             "text": ["true", "false", "foo", "bar", "qux"],
-            "int": [1, 0, 1, 0, 0],
+            "int": [1.0, 0.0, 1.0, 0.0, np.nan],
```

Comment:
Should we instead use `[1, 0, 1, 0, None]` with dtype "Int64"? I presume that's truer in spirit to the column that libcudf now constructs?

Reply:
That's a great question. 
Right now, when we encounter nulls in input integer column, we infer the column as float (WWPD). But with null support, we can keep it as "int64". But this will be another breaking behavior. I would prefer to make another PR for that breaking change.

Reply:
Thanks!